src/server/stores/user-store.js

```diff
@@ -31,10 +33,22 @@ function validate(password, user) {
  * @returns {Promise.<object|null>} Promise resolving to the user, or null if one cannot be found
  */
 export function get(email) {
-  if (!fakeStore.hasOwnProperty(email)) {
-    return Promise.resolve(null);
-  }
-  return Promise.resolve(Object.assign({}, fakeStore[email]));
+    // THE OLD CODE - still handy if you don't have a MySQL instance running...
+    //
+    //if (!fakeStore.hasOwnProperty(email)) {
+    //    return Promise.resolve(null);
+    //}
+    //return Promise.resolve(Object.assign({}, fakeStore[email]));
+    
+    return dbQuery("SELECT CONCAT(firstName,' ',lastName) AS name, photo AS avatar, email, password " +
+    "FROM users WHERE email=" + mysql.escape(email))
```

Comment:
We should really be using parameterized queries instead of relying on input escaping. https://github.com/mysqljs/mysql#preparing-queries